crates/federate/src/send.rs

```diff
@@ -108,7 +114,8 @@ impl SendRetryTask<'_> {
       .await
       .context("failed getting actor instance (was it marked deleted / removed?)")?;
 
-    let object = WithContext::new(object.clone(), FEDERATION_CONTEXT.deref().clone());
+    let object: DummyActivity = serde_json::from_value(object.clone())?;
+    let object = WithContext::new(object, FEDERATION_CONTEXT.deref().clone());
```

Comment:
it's a bit hard to tell what the performance implication of this is since this deserialization now has to happen for every instance instead of once for all instances.

Reply:
It should be quite fast as it only parses two fields and puts the rest from one `Value` into another. In release builds the compiler should produce similar code either way. Plus it simplifies the code for get_activity_cached().

Reply:
the main difference though is that it has to copy all the other values one for one from a serde_json Map to a hash map, so it scales by number of properties in the root object. serde json internally uses an incompatible map type so it has to make a new hash map.

doing this 1000 times for every activity _might_ be relevant for performance

Reply:
Right, changed `HashMap<String, Value>` to simply `Value`